utils/apps/cassandra/cassandra_liveness_cleanup.yml

```diff
@@ -0,0 +1,41 @@
+- name: Confirm that the cassandra liveness pod is running
```

Comment:
The steps for checking whether the liveness pod is running and subsequent steps to remove should all come from main logic.yaml. Lets keep on the cycleCompletion alone here. This way deletion will be handled (including rescue section) in the main logic.yaml

Other approach is to create a custom chaoslib that can create and remove liveness.